bundles/http_admin/test/test/http_websocket_tests.cc

```diff
@@ -222,7 +222,8 @@ TEST(HTTP_ADMIN_INT_GROUP, websocket_echo_test) {
 
     usleep(1000000); //Sleep for one second to let Civetweb handle the request
 
-    //Check if data received is the same as the data sent!
+    // ThreadSanitizer reports a potential data race on the data field, but can be ignored.
```

Comment:
Is there some way to instruct it to ignore this one, so that no race is reported?

Reply:
Yes. Two options:
* Run-time by adding a [supressions file](https://github.com/google/sanitizers/wiki/ThreadSanitizerSuppressions)
* Run-time by adding a blacklist to the [TSAN_OPTIONS](https://github.com/google/sanitizers/wiki/ThreadSanitizerFlags#compiler-flags-llvm-specific)

Reply:
Both only runtime, perhaps good to document somewhere. 

Since this is a test, can we add the file or options to the execution of it?
That way we do not run into CI errors due to this false positive.

Reply:
Thread sanitizer is currently not running on CI, so we're not getting this error at all. Even so, a case can be made for including a suppressions file.

Reply:
I'd like to have it yes.